src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -47,7 +47,13 @@ def handle_exception(ex):
             logger.error(ex)
         return 1
 
+    # Otherwise, unhandled exception. Direct users to create an issue.
+    logger.error("The command failed with an unexpected error. Here is the traceback:\n")
     logger.exception(ex)
+    logger.error("\nIf you think this is a bug, please visit: https://github.com/Azure/azure-cli to create an issue."
```

Comment:
I'd create an aka.ms link to guide people to our bug tracker. It'll do two things:
1. If we ever change where the product is hosted (because there was a nuclear apocalypse or something) we could update the link, and people with the old version of the product will still be able to file bugs.
1. We can track how many times people are clicking that link and creating issues.

Reply:
Thanks for the tip will do @marstr